src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/HtmlHelper.cs

```diff
@@ -957,6 +958,7 @@ protected virtual string GenerateDisplayText(ModelExplorer modelExplorer)
             ModelExplorer modelExplorer,
             string expression,
             object value,
+            string format,
```

Comment:
Don't think `HtmlHelper.GenerateHidden()` needs to change other than passing `format: null` to `_htmlGenerator.GenerateHidden()`.